AzureDb.Passwordless.MySql/AzureDb.Passwordless.MySql.csproj

```diff
@@ -3,13 +3,12 @@
 		<TargetFramework>netstandard2.0</TargetFramework>
 		<RepositoryUrl>https://github.com/felipmiguel/AzureDb.Passwordless</RepositoryUrl>
 	</PropertyGroup>
-
 	<ItemGroup>
-		<PackageReference Include="Azure.Identity" Version="1.8.2" />
-		<PackageReference Include="MySql.Data" Version="8.0.30.1" />
+	  <Compile Include="..\AzureDb.Passwordless.Core\TokenCredentialBaseAuthenticationProvider.cs" Link="TokenCredentialBaseAuthenticationProvider.cs" />
 	</ItemGroup>
 
 	<ItemGroup>
-		<ProjectReference Include="..\AzureDb.Passwordless.Core\AzureDb.Passwordless.Core.csproj" />
+		<PackageReference Include="Azure.Identity" Version="1.8.2" />
```

Comment:
if this is just for the samples reference, we should be able to remove this since the test project already has it